content/20.zksync-protocol/75.evm-interpreter/04.technical-details.md

```diff
@@ -118,16 +119,16 @@ in reserved pointer and does not change after these additional technical calls.
 
 Internally, deployment of an EVM contract from another EVM contract follows these steps:
 
-1. The emulator calls `precreateEvmAccountFromEmulator` in **ContractDeployer** for address derivation, increment of nonce and checks.
+1. The interpreter calls `precreateEvmAccountFromInterpreter` in **ContractDeployer** for address derivation, increment of nonce and checks.
 2. A **temporary bytecode hash** is assigned to mark the contract as an EVM contract.
-3. The emulator executes in **constructor** mode by interpreting the passed calldata as `initcode`.
-4. The final contract bytecode is returned by the emulator and stored.
+3. The interpreter executes in **constructor** mode by interpreting the passed calldata as `initcode`.
+4. The final contract bytecode is returned by the interpreter and stored.
 
 ## Summary of system contracts
 
 | **Component** | **Purpose** |
 |--------------|------------|
-| **EVM Emulator** | Interprets and executes EVM bytecode. |
+| **EVM Interpreter** | Interprets and executes EVM bytecode. |
```

Comment:
same as previous, let's keep this one as it was as it references the actual contract name